packages/wrangler/src/cloudchamber/build.ts

```diff
@@ -68,17 +69,20 @@ export async function constructBuildCommand(options: {
 }
 
 // Function for building
-export async function dockerBuild(options: { buildCmd: string }) {
-	const buildCmd = options.buildCmd.split(" ").slice(1);
-	const buildExec = options.buildCmd.split(" ").shift();
-	const child = spawn(String(buildExec), buildCmd, { stdio: "inherit" }).on(
-		"error",
-		(err) => {
-			throw err;
-		}
-	);
-	await new Promise((resolve) => {
-		child.on("close", resolve);
+export async function dockerBuild(options: {
```

Comment:
nit: I think we shouldnt mark the function async as we are already returning a promise and we dont call await